### PR TITLE
Implement method for getting analysis metadata

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -168,7 +168,23 @@ def parse_python_solver_name(solver_name: str) -> dict:
...
+            self._session.query(PackageExtractRun)
+            .filter(PackageExtractRun.analysis_document_id == analysis_document_id)
+            .with_entities(
+                PackageExtractRun.datetime,
+                PackageExtractRun.analysis_document_id,
+                PackageExtractRun.package_extract_name,
+                PackageExtractRun.package_extract_version
+            )
+        )
+        query_result = query.one()
```
Comment (pacospace) [inline thoth/storages/graph/postgres.py]: Don't we add an error if the query is empty? https://github.com/thoth-station/storages/blob/39206123de88916cbd4143b45b16796cad2bd4bc/thoth/storages/graph/dgraph.py#L303

Comment (fridex) [reply thoth/storages/graph/postgres.py]: `one()` raises an error if no results are found. We should map it to our error though.

Comment (pacospace) [reply thoth/storages/graph/postgres.py]: do you want to change it?

Comment (fridex) [reply thoth/storages/graph/postgres.py]: Adjusted in aaf5f0f :+1:

**Pull request merged**